### ISSUE TITLE
Explore possible Cython speedups

### ISSUE THREAD

**Issue by schmidtfa** [schmidtfa/pyrasa#25]

**Comment by schmidtfa** [comment]
@thht what do you think about trying out [mypyc](https://mypyc.readthedocs.io/en/latest/introduction.html#why-mypyc)  instead of cython for speed explorations?

**Comment by thht** [comment]
Seems like being worth a shot. Even though it's alpha....

**Comment by schmidtfa** [comment]
After working on this for a day or two I realised that this is probably not the best idea for now. While I managed to get some speed-ups the performance increases weren't super crazy.. If we really want high-performance I think its best to implement sth like an irasa_high_speed module in the future instead of trying to accomplish that with the current implementation. The reason for this is that with the current irasa we really focused on making the tool very flexible such that users have a lot of power in how they can specify their models. I think for a highspeed cython implementation its probably better to have a lot of things preset from the getgo and to not have so much flexibility in between. So I would abandon this for now and build a seperate module for version 2 (if we think its useful).
